TaskManager/Modules/Add Button/View/AddTaskViewController.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  AddTaskViewController.swift
+//  TaskManager
+//
```

Comment:
lets create a builder for add module. And the module name should me AddTask. The module names usually mean the view that is implemented. In your case you have the AddTask screen so AddTask name for the module is ok, or Home screen so Home module name is ok